platforms/hyperledger-fabric/configuration/samples/network-fabric-add-organization.yaml

```diff
@@ -228,7 +228,8 @@ network:
         - peer:
           name: peer0          
           type: anchor    # This can be anchor/nonanchor. Atleast one peer should be anchor peer.         
-          gossippeeraddress: peer0.manufacturer-net:7051 # Internal Address of the other peer in same Org for gossip, same peer if there is only one peer          
+          gossippeeraddress: peer0.manufacturer-net:7051 # Internal Address of the other peer in same Org for gossip, same peer if there is only one peer 
+          cli: enabled         
```

Comment:
our samples should have this as disabled by default. Also add a comment about what this does.